telemetry-impls/_github-actions-job-info/action.yml

```
@@ -55,5 +56,8 @@ runs:
             throw "Telemetry values were not available. Please see debug logs for more info."
           }
           core.setOutput("job_info_json", this_job);
-    - shell: bash
-      run: echo "${{toJSON(steps.get-job-info.outputs.job_info_json)}}" > job_info.json
\ No newline at end of file
+          try {
+            fs.writeFileSync('job_info.json', JSON.stringify(this_job));
+          } catch(err) {
+           console.error(err)
```

```suggestion
          } catch(err) {
           console.error(err)
           throw(err)
```

I assume this error should still `throw`, right?

If so, you can probably just omit the `try`/`catch` block entirely. In the event of an error, this would throw an unhandled exception which is pretty much equivalent to logging to `stderr` followed by a `throw`